.github/workflows/ansible-release.yml

```diff
@@ -0,0 +1,84 @@
+name: ansible-release
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example : 11.1.0
+        required: true
+      ansible-major-version:
+        description: Example 11
+        required: true
```

Comment:
This input isn't really needed. Instead, calculate the major version from `ansible-version` and set that as a step/job output for reuse in runtime.